### DISCUSSION TITLE
How to get files length in QUploader

### DISCUSSION THREAD

**Discussion by jiatiejun** [quasarframework/quasar#9295] [General - Components / Directives / etc]
I want to get files lengh in QUploader
but @added , @removed only is a part of everytimes
i want to get current total files length

how to get ?

**Comment by metalsadman** [comment]
@jiatiejun you can use a ref on your quploader, then check the `files` property. ie `this.$refs.uploaderRef.files`, either you check it on your `@added`, `@removed` handlers or thru a `watcher`.

https://codepen.io/metalsadman/pen/ZEepWEJ?editors=1010
